repos/spack_repo/builtin/packages/rdkit/package.py

```diff
@@ -73,7 +73,7 @@ class Rdkit(CMakePackage):
         depends_on("coordgen", when="+coordgen")
         depends_on("maeparser", when="+maeparser")
         depends_on("eigen@3:", when="+descriptors3d")
-        depends_on("python@3:", when="+python")
+        depends_on("python@:3.11", when="+python")
```

Comment:
This is for `2022_09_5:` which means `2022_09_5` and above. Shouldn't this constraint be for `:2022_09_5`?